kvmagent/kvmagent/plugins/tensorfusion/service.py

```
@@ -42,11 +42,38 @@ def initialize(self):
+
+        # Kill orphan workers whose VM is no longer running in libvirt.
+        alive = []
+        for w in running:
+            if self._is_vm_running(w.vm_uuid):
+                alive.append(w)
+            else:
+                logger.warn('TensorFusionService: killing orphan worker %s '
+                            '(pid=%d, vm=%s not running)' % (w.device_uuid, w.pid, w.vm_uuid))
+                try:
+                    self._executor.stop(w)
+                except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)